test/ao/sparsity/test_composability.py

```diff
@@ -75,6 +76,7 @@ class TestComposability(TestCase):
     # This test checks whether performing quantization prepare before sparse prepare
     # causes any issues and verifies that the correct observers are inserted and that
     # the quantized model works as expected
+    @unittest.skipIf(IS_S390X, "Currently fails on s390x")
```

Comment:
Echo @malfet's point earlier, is this possible to just use `skipIfS390X` here (and all other places too)?


```suggestion
    @skipIfS390X
```